wit-manifest.json

```diff
@@ -1,31 +1,21 @@
 [
     {
-        "commit": "69c73cfe30525093b4d6d7acec9d5fb854209d13",
+        "commit": "7bcafebc0ea9c975cbcd057ec97c78bac82249ef",
         "name": "api-generator-sifive",
         "source": "git@github.com:sifive/api-generator-sifive.git"
     },
     {
-        "commit": "fcff9cc0ac5dbd92eb5da5d4b855de22c402c302",
+        "commit": "c1dee8234c23c8fc454108e59ecba20987f95cde",
         "name": "sifive-blocks",
         "source": "git@github.com:sifive/sifive-blocks.git"
     },
-    {
-        "commit": "e1aa5f3f5c0cdeb204047c3ca50801d9f7ea25f1",
-        "name": "chisel3",
-        "source": "git@github.com:freechipsproject/chisel3.git"
-    },
```

Comment:
removing firrtl dep is fine, but I think there should still be a chisel3 dependency, because we directly depend on it in the scala code?

Reply:
Why isn't relying on the transitive dependency from rocket-chip sufficient? I am quite confident in practice that rocket-chip is always going to depend on chisel3 itself, and at the moment i don't need a chisel3 newer than the one rocket-chip has?

Reply:
same reason as https://github.com/sifive/block-pio-sifive/pull/70#discussion_r421162562. but yeah, not really a big deal and doesn't have to block this PR.

Reply:
i put it back